ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -689,6 +691,57 @@ -(void)saveDocument:(NSDictionary<NSString *, id> *)arguments resultToken:(Flutt
     
 }
 
+-(void)getPageCropBox:(NSDictionary<NSString *,id> *)arguments resultToken:(FlutterResult)result
```

Comment:
Why does this method exist on both PdftronFlutterPlugin and FlutterDocumentView?

Reply:
For Native Module and Widget? There are two ways to open a viewer/editor